generate_user_guides.sh

```diff
@@ -1,32 +1,41 @@
 #!/bin/bash
+set -v
+
+mkdir -p docs/user_guides/rust/
+mkdir -p docs/user_guides/bevy_plugin/
+mkdir -p docs/user_guides/javascript/
+
+# NOTE: injecting all code here is suboptimal because we're copying it 3 times
+#       and parsing all those multiple times. Ideally we would inject the code at the last step.
+find docs/user_guides/templates/ -type f -print0 | OUTPUT_FOLDER=docs/user_guides/templates_injected/ xargs -0 ./inject_code_in_user_guides.sh
 
 cp -r docs/user_guides/templates/* docs/user_guides/rust/.
 cp -r docs/user_guides/templates/* docs/user_guides/bevy_plugin/.
 cp -r docs/user_guides/templates/* docs/user_guides/javascript/.
 
 find docs/user_guides/bevy_plugin/ -type f -print0 | xargs -0 gawk -i inplace -v RS='<rapier>[^<]*</rapier>' -v ORS= '1'
-find docs/user_guides/bevy_plugin/ -type f -print0 | xargs -0 gsed -i '/<rapier>/,/<\/rapier>/d'
```

Comment:
these should be reverted

Reply:
Still I'm curious if it's needed ; I tested from windows and mac and I didn't notice any major difference (but maybe the devil is in the details?)

Reply:
Let’s keep your changes. Maybe there is something weild going on my end.